backend/src/main/java/org/eclipse/tractusx/puris/backend/stock/logic/service/ReportedMaterialItemStockService.java

```diff
@@ -21,6 +21,10 @@
 package org.eclipse.tractusx.puris.backend.stock.logic.service;
 
 import lombok.extern.slf4j.Slf4j;
+
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
```suggestion
import java.util.ArrayList;
import java.util.List;
```
I would not introduce new blanks in the imports